third_party/s2n-bignum/include/s2n-bignum_aws-lc.h

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
`s2n-bignum` is a subtree. Can it figure out having an extra file locally when merging from upstream?

Reply:
We filter the changes from s2n-bignum only to arm and x86 directories so the git subtree magic should ignore include dir.